node/src/main/scala/com/wavesplatform/api/http/AddressApiRoute.scala

```diff
@@ -287,19 +286,22 @@ case class AddressApiRoute(
       pass
   }
 
-  private def accountData(address: Address)(implicit sc: Scheduler) =
-    commonAccountsApi
-      .dataStream(address, None)
-      .toListL
-      .runAsyncLogErr
-      .map(data => Source.fromIterator(() => data.sortBy(_.key).iterator.map(Json.toJson[DataEntry[_]])))
+  private def accountData(address: Address)(implicit m: ToResponseMarshaller[Source[JsValue, NotUsed]]) = {
+    routeTimeout.execute(
+      commonAccountsApi
+        .dataStream(address, None)
+        .toListL
+        .map(data => Source.fromIterator(() => data.sortBy(_.key).iterator.map(Json.toJson[DataEntry[?]])))
+    )(_.runAsyncLogErr(_))
+  }
 
-  private def accountData(addr: Address, regex: String)(implicit sc: Scheduler) =
-    commonAccountsApi
-      .dataStream(addr, Some(regex))
-      .toListL
-      .runAsyncLogErr
-      .map(data => Source.fromIterator(() => data.sortBy(_.key).iterator.map(Json.toJson[DataEntry[?]])))
+  private def accountData(addr: Address, regex: String)(implicit m: ToResponseMarshaller[Source[JsValue, NotUsed]]) =
```

Comment:
The only difference of these two `accountData` methods is `regex: String`. How about:
```scala
private def accountData(addr: Address, regex: Option[String] = None)(implicit m: ToResponseMarshaller[Source[JsValue, NotUsed]])
```
?